### PR TITLE
changed output variable to checks.stageId

### PR DESCRIPTION
**Pull request by v-hinti**
Service now failing at update change request due mis match stage id's

### CODE REVIEW

**Comment by pavanvamsi3** [comment]
We're using `checks.stageId` in both POST and GET requests now?

**Comment by v-hinti** [comment]
> We're using `checks.stageId` in both POST and GET requests now?

yes in both  i have updated

please find get request 

image

please find  post request
image

**Comment by v-hinti** [comment]
i think we have to  release CreateAndQueryChangeRequestV3 with effecting existing customers

**Comment by v-hinti** [comment]
@tauhid621 , @pavanvamsi3  time being don't approve the PR just review this PR i need to perform some tests in dev fabric.

File: Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/CreateAndQueryChangeRequestV2/task.json [hunk 1]
```diff
@@ -360,7 +360,7 @@
         {
           "RequestInputs": {
             "EndpointId": "$(ServiceNowConnection)",
-            "EndpointUrl": "$(endpoint.url)/api/now/table/change_request?x_mioms_azpipeline_stage_id=$(system.stageId)&x_mioms_azpipeline_build_id=$(build.buildId)&x_mioms_azpipeline_stage_attempt=$(checks.stageAttempt)&sysparm_fields=number,sys_id,correlation_id",
+            "EndpointUrl": "$(endpoint.url)/api/now/table/change_request?x_mioms_azpipeline_stage_id=$(checks.stageId)&x_mioms_azpipeline_build_id=$(build.buildId)&x_mioms_azpipeline_stage_attempt=$(checks.stageAttempt)&sysparm_fields=number,sys_id,correlation_id",
```
Comment (tauhid621) [inline Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/CreateAndQueryChangeRequestV2/task.json:363]: Would be great if you can add in the description what the new stageId will look like.

**Comment by pavanvamsi3** [comment]
@v-hinti This is a bug fix, we shouldn't release a new major version.

**Comment by v-hinti** [comment]
@pavanvamsi3, @tauhid621 service now rerun stage fixed here please check and update

**Comment by pavanvamsi3** [comment]
@v-hinti - there's unresolved conversation that @tauhid621 had mentioned.

**Review by pavanvamsi3 — APPROVED** [review]

**Review by tauhid621 — APPROVED** [review]

**Pull request merged**